magento/magento2
# Enable sort order to product Custom options

### Description (*)
Change sort order of custom option values over several pages in product edit view

### Related Pull Requests
 Enabled the sort order field to custom option and custom option values.

### Fixed Issues (if relevant)
1. Fixes #28153 

### Manual testing scenarios (*)

1. Create a new simple product in the backend
3. Add a custom option of type drop-down
4. Add 21 values, the last value appears on a new page.
5. Try to move the last value to another position on the first page.


### Contribution checklist (*)
